charts/dex/templates/service.yaml

```diff
@@ -50,6 +50,9 @@ spec:
     {{- end }}
     - name: telemetry
       port: 5558
+      {{- if and (or (eq .Values.service.type "NodePort") (eq .Values.service.type "LoadBalancer")) >Values.service.ports.telemetry.nodePort }}
```

Comment:
I'm not familiar with what `>` means in go templating or helm — is this the same as hasKey?